build_tools/python/benchmark_suites/iree/cuda_benchmarks.py

```diff
@@ -24,7 +24,10 @@ class Linux_CUDA_Benchmarks(object):
   SM_80_COMPILE_CONFIG = iree_definitions.CompileConfig.build(
       id=unique_ids.IREE_COMPILE_CONFIG_LINUX_CUDA_SM80_DEFAULTS,
       tags=["default-flags"],
-      compile_targets=[SM_80_GPU_TARGET])
+      compile_targets=[SM_80_GPU_TARGET],
+      extra_flags=[
+          "--iree-codegen-llvmgpu-use-mma-sync",
```

Comment:
These are the benchmarks with default flags, so it looks a little bit weird to set flags here. What is this flag doing and is it possible to make this option enabled by default?

Reply:
The plan is to enable this functionality by default but we are currently running into some compiler errors for some models: https://github.com/openxla/iree/issues/13075

Reply:
Maybe add a TODO with reason?

Reply:
Added a comment

Reply:
No we don't want extra flags in "default flags" configs. That is the whole reason for them. I can dig up the PR where I created this split. If this still needs some special flags to run then it should be in "experimental flags".

Reply:
That's fair. I'll let the relevant parties know that it won't be enabled by default until the compiler issues are fixed.